src/runtime/Makefile

```diff
@@ -47,6 +47,9 @@ PROJECT_ORG = $(PROJECT_TAG)
 PROJECT_URL = https://github.com/$(PROJECT_ORG)
 PROJECT_BUG_URL = $(PROJECT_URL)/kata-containers/issues/new
 
+# @RUNTIME_NAME@ should be replaced with the target in generated files
+RUNTIME_NAME = $(TARGET)
```

Comment:
@c3d, TARGET is set down below, line 65 on your file. We would need the RUNTIME_NAME to be set **after** the TARGET variable is set, wouldn't we?

Reply:
Not necessary with makefile variables. `=` assigned the value `$(TARGET)`, and that value is only expanded when you use it.

Reply:
@fidencio That being said, if you find it confusing, it's easy to fix.

Reply:
I pushed a new version assuming it improves readability.

Reply:
Aha! I do find it confusing, indeed!

Thanks a lot for moving it down the path!